src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -26,70 +26,70 @@ public void asObservableList_modifyList_throwsUnsupportedOperationException() {
     @Test
     public void isSamePerson() {
         // same object -> returns true
-        assertTrue(ALICE.isSamePerson(ALICE));
+        assertTrue(ANCHOVIES.isSamePerson(ANCHOVIES));
 
         // null -> returns false
-        assertFalse(ALICE.isSamePerson(null));
+        assertFalse(ANCHOVIES.isSamePerson(null));
 
         // different phone and email -> returns false
-        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).build();
-        assertFalse(ALICE.isSamePerson(editedAlice));
+        Person editedAlice = new PersonBuilder(ANCHOVIES).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).build();
+        assertFalse(ANCHOVIES.isSamePerson(editedAlice));
 
         // different name -> returns false
```

Comment:
Maybe the occurrences of name in comments should be replaced with description?

Reply:
Changed all comments with name to description.